QEfficient/utils/_utils.py

```diff
@@ -42,7 +42,7 @@ def hf_download(
     if cache_dir is not None:
         os.makedirs(cache_dir, exist_ok=True)
 
-    max_retries = 5
+    max_retries = Constants.MAX_RETRIES
     retry_count = 0
     while retry_count < max_retries:
```

Comment:
directly use 
```retry_count<constants.MAX_RETRIES``` instead of redefining on line 45

Reply:
Okay, I will do that.